apps/client/src/features/viewers/minimal-timer/MinimalTimer.tsx

```diff
@@ -133,14 +133,18 @@ export default function MinimalTimer(props: MinimalTimerProps) {
   const isNegative = (time.current ?? 0) < 0;
   const showFinished = isNegative && !userOptions?.hideOvertime;
   const showEndMessage = time.current < 0 && general.endMessage && !hideEndMessage;
+  const countUp =
+    time.duration - time.current > time.duration ? time.duration * -1 + time.current : time.duration - time.current;
+  const timer =
+    time.countdownStyle === 'count-down' ? time.current : time.countdownStyle === 'count-up' ? countUp : time.clock;
```

Comment:
I would prefer we skip the nested ternaries
Once we have our function, you could extract this into an utility function that receives the countdown style and the object of timers and returns the appropriate timer